docker-compose.yml

```diff
@@ -146,9 +147,38 @@ services:
     volumes: []
     environment:
       - ELASTIC_PASSWORD=serxdfcghjfc
-    networks: [ datalayer-elasticsearch, search-datalayer ]
+    networks: [ datalayer-elasticsearch, pgsync-elasticsearch, search-elasticsearch ]
```

Comment:
This is clearly the wrong networks, but if everything works we can fix this later

Reply:
gotcha